src/badguy/badguy.cpp

```diff
@@ -590,9 +611,17 @@ BadGuy::set_state(State state_)
 bool
 BadGuy::is_offscreen() const
 {
+  Vector dist;
+  if (EditorActive()) {
+    Camera *cam = Sector::current()->camera;
+    dist = cam->get_center() - bbox.get_middle();
+  } else {
+    Player* player = get_nearest_player();
+    if (!player) return false;
+    dist = player->get_bbox().get_middle() - bbox.get_middle();
+  }
   Player* player = get_nearest_player();
```

Comment:
Those two lines seem to be a duplicate from l619 f. Can't we avoid this minor code duplication?